tcms/core/views.py

```diff
@@ -66,6 +72,31 @@ def server_error(request):  # pylint: disable=missing-permission-required
     return http.HttpResponseServerError(template.render({}, request))
 
 
+class SetupView(TemplateView):
+
+    template_name = "setup.html"
+
+    def get_context_data(self, **kwargs):
+        # Check for non-applied migrations
+        executor = MigrationExecutor(connections[DEFAULT_DB_ALIAS])
+        plan = executor.migration_plan(executor.loader.graph.leaf_nodes())
```

Comment:
You don't need to check explicitly for migrations. Chances are this check could fail anyway b/c we don't have a functioning database (e.g. the table in which Django records it's migrations is missing). Apply the migrations unconditionally. They will be skipped if already applied anyway.

If the problem is more significant then we can't really help anyway.